sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -1130,7 +1130,7 @@ class AdaptiveQueryExecSuite
         SQLConf.AUTO_BROADCASTJOIN_THRESHOLD.key -> "-1",
         SQLConf.SHUFFLE_PARTITIONS.key -> "100",
         SQLConf.SKEW_JOIN_SKEWED_PARTITION_THRESHOLD.key -> "800",
-        SQLConf.ADVISORY_PARTITION_SIZE_IN_BYTES.key -> "1000") {
+        SQLConf.ADVISORY_PARTITION_SIZE_IN_BYTES.key -> "900") {
```

Comment:
Is this also due to `LZ4`'s behavior difference?

Reply:
Yes, I believe so. I didn't dig as deeply into this test as I did with the CoalesceShufflePartitionsSuite but the partitioning is always affected by the different lz4 compression behavior between big and little endian. The only thing needed for this test to pass is a change to a partition size parameter which is definitely consistent with the lz4 compression behavior.